subcommands/keys/ca_renewal_start.go

```diff
@@ -0,0 +1,92 @@
+package keys
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/foundriesio/fioctl/client"
+	"github.com/foundriesio/fioctl/subcommands"
+	"github.com/foundriesio/fioctl/x509"
+)
+
+func init() {
+	cmd := &cobra.Command{
+		Use:   "start <New PKI Directory> <Old PKI Directory>",
+		Short: "Start a root of trust renewal for your factory PKI",
+		Run:   doStartCaRenewal,
+		Args:  cobra.ExactArgs(2),
+		Long: `Start a root of trust renewal for your factory PKI.
+
+This command does a few things:.
+1. First, it generates a new root of trust for your factory.
+2. Second, it cross-signs a new root of trust using an old root of trust to prepare a CA renewal bundle.
+   This CA renewal bundle is compliant with the EST standard, necessary for a secure root CA update on devices.
+3. Finally, it uploads the CA renewal bundle to the backend API for validation and storage.
```

Comment:
not sure where to ask in the PR, but I'm wondering if there's going to wind up being an easy way to check "if in progress" and also make sure two different people don't try this at the same time?

Reply:
I planned to add a kind of `review|status` command similar to the one we have for TUF rotations.
Just haven't got idea about its design yet... and it is highly dependent onto what I do on the API side.